.github/workflows/enonic-gradle.yml

```diff
@@ -3,25 +3,123 @@ name: Gradle Build
 on: [ push ]
```

Comment:
Why was this file changed? 
This is not part of the angular change.

Reply:
This is also not an up to date build and deploy action.

Reply:
My bad. I merged the master intro this branch in order to solve the conflict. Now I've done it properly. @poi33 please check last commit.